decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -1,3 +1,8 @@
+<% content_for(:meta_description){ @proposal.body } %>
```

Comment:
Sorry for trolling, but `content_for` accepts strings as a parameters as well, in case you want to use it:

```ruby
content_for :meta_description, @proposal.body
```

This way, you don't risk injecting template elements. Anyway, not mandatory ;)